autoPyTorch/pipeline/components/training/losses.py

```diff
@@ -11,16 +11,17 @@
 from autoPyTorch.constants import BINARY, CLASSIFICATION_TASKS, CONTINUOUS, MULTICLASS, REGRESSION_TASKS, \
     STRING_TO_OUTPUT_TYPES, STRING_TO_TASK_TYPES, TASK_TYPES_TO_STRING
 
+
 losses = dict(classification=dict(
     CrossEntropyLoss=dict(
-        module=CrossEntropyLoss, supported_output_type=MULTICLASS),
```

Comment:
Can you change those names to "MultiClass" and "Binary"? Plus, it would be better to define typing variable in the file that defines MULTICLASS and BINARY. e.g. SupportedOutputTypes = Union[MultiClass, Binary]

Reply:
okay, should I change the supported output types also or you'll include it in your PR as well?

Reply:
I think it would be better to do it by yourself because you are more familiar with the related codes

Reply:
okay that makes sense

Reply:
I think these supported output types is a property of the particular loss. for example, if we had our own loss functions, this information will go to the get_properties function and hence for me it does not make sense to create supported output types as a typing variable.

Reply:
> these supported output types are a property of the particular loss
Hum, that is true. But at the same time, we would not know the supported types explicitly from the codes, so let's make sure we can know the supported type explicitly (doc-string or class function such as CrossEntropyLoss.is_output_type_supported() etc...)

Reply:
yes I'll add doc string for this file.